compiler/rustc_codegen_ssa/src/mir/block.rs

```diff
@@ -1224,7 +1249,19 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
             }
 
             mir::TerminatorKind::Goto { target } => {
-                helper.funclet_br(self, bx, target, mergeable_succ())
+                let add_tapir_metadata = if self.parallel_back_edges.contains(bb) {
+                    AddParallelLoopMetadata::True
+                } else {
+                    AddParallelLoopMetadata::False
+                };
+                // NOTE(jhilton): we attach the metadata to the parallel loop back edge.
```

Comment:
We probably don't need to leave this comment, but it's not hurting anyone here.

Reply:
Resolved in [500cb9b](https://github.com/aleph-oh/rust/pull/18/commits/500cb9b34e3a3c2710d99dc57408cd6de029071b).